docs/Homebrew-on-Linux.md

```diff
@@ -79,7 +79,7 @@ To install build tools, paste at a terminal prompt:
 
 Homebrew can run on 32-bit ARM (Raspberry Pi and others) and 64-bit ARM (AArch64), but as they lack binary packages (bottles) they are unsupported. Pull requests are welcome to improve the experience on ARM platforms.
 
-You may need to install your own Ruby using your system package manager, a PPA, or `rbenv/ruby-build` as we no longer distribute a Homebrew Portable Ruby for ARM.
+You may need to install your own Ruby using your system package manager, a PPA, or `rbenv/ruby-build` as we no longer distribute a Homebrew Portable Ruby for ARM. You might also need to install the `cpan` package.
```

Comment:
I know this was closed anyway but: I really don't want the ARM documentation here to grow over time. It's unsupported: that's all we really need to say here.